monitor/tests/mocks/mock_redis.py

```diff
@@ -34,24 +37,26 @@ def __init__(self):
     Returns:
         None
     """
+
     def rpush(self, metric_queue, metric):
-        if self.map.get(metric_queue) == None:
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR?